internal/app/api/v1/tests.go

```diff
@@ -220,7 +227,7 @@ func (s TestKubeAPI) executeTestStep(ctx context.Context, testName string, step
 	case testkube.TestStepTypeExecuteScript:
 		executeScriptStep := step.Execute
 		options, err := s.GetExecuteOptions(executeScriptStep.Namespace, executeScriptStep.Name, testkube.ExecutionRequest{
-			Name: fmt.Sprintf("%s-%s", testName, executeScriptStep.Name),
+			Name: fmt.Sprintf("%s-%s-%s", testName, executeScriptStep.Name, rand.String(5)),
```

Comment:
Just in case, random there is a math random and it's not unique

Reply:
yeah I'm doing seed in common.go :) in rand package